go.mod

```diff
@@ -20,7 +20,6 @@ require (
 	github.com/spf13/viper v1.14.0
 	github.com/strangelove-ventures/packet-forward-middleware/v3 v3.1.1
 	github.com/stretchr/testify v1.8.1
-	github.com/tendermint/tendermint v0.34.24
```

Comment:
No need to remove here

Reply:
Why? I though that we don't need to directly import TM.

Reply:
We actually do, other dependencies require it.

Reply:
This was the go mod action